packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -209,10 +209,11 @@ function validateDangerousTag(tag) {
 
 const styleNameCache = {};
 const processStyleName = function(styleName) {
+  const isCustomProperty = styleName.indexOf('--') === 0;
```

Comment:
You likely meant to put this after the conditional return so it doesn't execute for every style name.

Reply:
thanks, resolved!